src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -128,6 +150,12 @@ public ObservableList<Person> getPersonList() {
         return persons.asUnmodifiableObservableList();
     }
 
+    @Override
+    public ObservableList<Property> getPropertyList() {
+        System.out.println(properties.asUnmodifiableObservableList().size());
```

Comment:
Is this for testing? Might need to delete it later to keep the code neat.

Reply:
oh yes, i was checking where the code was throwing me a NullPointerException.